client/test/commands/teardownFabricRuntime.test.ts

```diff
@@ -67,11 +79,28 @@ describe('teardownFabricRuntime', () => {
     });
 
     it('should teardown a Fabric runtime', async () => {
```

Comment:
What's the difference between this test and the test on line 92?

Reply:
in the first test the gateway panel isn't connected so doesn't need to disconnect, whereas in the second test it is connected to a local runtime so therefore needs to disconnect